torch/csrc/jit/codegen/cuda/ir_iostream.cpp

```diff
@@ -170,8 +167,7 @@ void IRPrinter::handle(const Float* const f) {
     os << "f" << f->name();
   } else {
     os << "float("
-       << std::setprecision(
-              std::numeric_limits<Float::ScalarType>::max_digits10)
+       << std::setprecision(std::numeric_limits<float>::max_digits10)
```

Comment:
Looks like it's reverting previous PR for no good reason.

Reply:
Reverted, thanks.